README.md

```diff
@@ -76,6 +76,13 @@ The following are all _required_.
         version: 'v1.0.1'
     ```
 
+- Create a new Sentry release for [Self-Hosted Sentry](https://develop.sentry.dev/self-hosted/)
+    ```yaml
+    - uses: getsentry/action-release@v1
+      env:
+        SENTRY_URL: https://sentry.example.com/
+    ```
+
```

Comment:
Looks great! Do you mind also adding to the 'Environment Variables' section a part that mentions `SENTRY_URL`? Here's my suggestion, though if you have a clearer way of conveying it, that works too :+1:

```markdown
#### Environment Variables
The following are all _required*_.
|name|description|
|---|---|
|`SENTRY_AUTH_TOKEN`|Authentication token for Sentry. See [installation](#create-a-sentry-internal-integration).|
|`SENTRY_ORG`|The slug of the organization name in Sentry.|
|`SENTRY_PROJECT`|The slug of the project name in Sentry.|
|`SENTRY_URL`*|The URL used to connect to Sentry, defaults to `https://sentry.io`.<br/>(Only required for [Self-Hosted Sentry](https://develop.sentry.dev/self-hosted/))|
```

Reply:
@leeandher I have added a description at 1560185. What do you think?